tests/conftest.py

```diff
@@ -582,8 +584,8 @@ def nbrhosts(enhance_inventory, ansible_adhoc, tbinfo, creds, request):
         logger.info("No VMs exist for this topology: {}".format(tbinfo['topo']['properties']['topology']))
         return devices
 
-    for k, v in list(tbinfo['topo']['properties']['topology']['VMs'].items()):
-        vm_name = vm_name_fmt % (vm_base + v['vm_offset'])
+    def initial_neighbor(neighbor_name, vm_name):
+        logger.info(f"nbrhosts started: {neighbor_name}_{vm_name}")
```

Comment:
@yejianquan can you change this to `debug` level? we see too many logs when running on T2, T2 has lots of neighbors

Reply:
@wenyiz2021 Let's keep it for now, there's still room for performance tuning on T2 neighbor initialization.
These log helps to analyze the time spending.